supertokens_python/utils.py

```diff
@@ -179,3 +182,14 @@ def frontend_has_interceptor(request: BaseRequest) -> bool:
 
 def deprecated_warn(msg: str):
     warnings.warn(msg, DeprecationWarning, stacklevel=2)
+
+
+def handle_httpx_client_exceptions(e: Exception, input_: Dict[str, Any]):
+    if isinstance(e, HTTPStatusError):
+        res: Response = e.response  # type: ignore
+        log_debug_message("Error status: %s", res.status_code)  # type: ignore
+        log_debug_message("Error response: %s", res.json())
+    else:
+        log_debug_message("Error: %s", e)
+        log_debug_message("Logging the input:")
+        log_debug_message("%s", json.dumps(input_))
```

Comment:
why are you logging the input in only one case?

Reply:
Fixed.